corpus_descriptions_test/portage_pkg.list

```diff
@@ -0,0 +1,4135 @@
+x11-libs/vte
```

Comment:
There should be a script to generate this and this file should not be checked in.